submissions/amooChristianah/database/TransportationCompany.sql

```diff
@@ -0,0 +1,44 @@
+create DATABASE TRANSPORTATION;
```

Comment:
This supposes to be for your next task. Kindly remove it

Reply:
done. kindly review